types/react-navigation/index.d.ts

```diff
@@ -10,6 +10,7 @@
 //                 charlesfamu <https://github.com/charlesfamu>
 //                 Tim Wang <https://github.com/timwangdev>
 //                 Qibang Sun <https://github.com/bang88>
+//                 Sergei Butko: <https://github.com/svbutko>
```

Comment:
Also adding your name here, says you agree to be a maintainer for these typings. So you'll be pinged for code reviews and issues. Just making sure you know what you're doing.

Reply:
Yes, I know about this, just didn't know about TS versioning. Every project depends on this library and I'll try to contribute to it and try to review the code.

Reply:
Great 👍